### PR TITLE
GPU: Added some memory fill tests

### PR DESCRIPTION
**Pull request by Subv**
With https://github.com/citra-emu/citra/pull/646 , Citra passes all of these

### CODE REVIEW

File: source/tests/gpu/memoryfills.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    u8* buffer = (u8*)vramAlloc(0x400);
+    
+    Test(tag, "Fill24bits", Fill24bits(buffer), true);
+    Test(tag, "Fill32bits", Fill32bits(buffer), true);
+    
+    vramFree(buffer);
+}
+
+}
+}
```
Comment (archshift) [inline source/tests/gpu/memoryfills.cpp]: newline

File: source/tests/gpu/memoryfills.h [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace GPU {
+namespace MemoryFills {
+
+void TestAll();
+
+}
+}
```
Comment (archshift) [inline source/tests/gpu/memoryfills.h]: newline

File: source/tests/gpu/gpu.cpp [hunk 3]
```diff
@@ -9,6 +10,7 @@ void TestAll() {
     GPU_Init(nullptr);
     
     DisplayTransfer::TestAll();
+    MemoryFills::TestAll();
 }
 
 }
```
Comment (archshift) [inline source/tests/gpu/gpu.cpp]: newline

File: source/tests/gpu/memoryfills.cpp [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    
+    return true;
+}
+
+void TestAll() {
+    const std::string tag = "MemoryFills";
+    
+    u8* buffer = (u8*)vramAlloc(0x400);
+    
+    Test(tag, "Fill24bits", Fill24bits(buffer), true);
```
Comment (archshift) [inline source/tests/gpu/memoryfills.cpp]: Capitalize "Bits"?

File: source/tests/gpu/memoryfills.cpp [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    Test(tag, "Fill32bits", Fill32bits(buffer), true);
```
Comment (archshift) [inline source/tests/gpu/memoryfills.cpp]: Again on capitalization

**Comment by archshift** [comment]
Other than that, LGTM

**Comment by Subv** [comment]
Addressed all comments, should be good to go

**Comment by bunnei** [comment]
LGTM

**Pull request merged**